app.js

```diff
@@ -195,27 +195,24 @@ function creatPopup(imageSource) {
   />
   <div class="modal-content">
   <p class="modal-texts">
-    lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam
-    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Officiis voluptatibus error veritatis
-    ad minima nesciunt harum sed, fugiat eum quis, architecto impedit inventore nam. Assumenda repellendus
-    <br>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem 
-    Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown
-     printer took a galley of type and scrambled it to make a type specimen book. It has survived
-      not only five centuries, but also the leap into electronic typesetting, remaining essent
+    ${description} 
+    standard dummy text ever since the 1500s, when an unknown
+    printer took a galley of type and scrambled it to make a type specimen book. It has survived
+    not only five centuries, but also the leap into electronic typesetting, remaining essent
     iste cum consequatur, unde porro laboriosam velit. Consectetur repellendus
     illo iure reiciendis repudiandae voluptatum velit, placeat a culpa ut nemo aut suscipit? Pariatur, a.
   </p>
```

Comment:
- [ ] [OPTIONAL] I understand you're trying to make the description a bit longer but I'd suggest you just stick to the description of the project object as having this dummy text makes your portfolio look unprofessional even in the early development stages.